rune/libenclave/intelsgx/aesmd.go

```diff
@@ -100,9 +100,12 @@ func DumpTargetInfo(targetInfo []byte) error {
 	return nil
 }
 
-func DumpLocalReport(report []byte) error {
+/* The report can be used as a separate structure or as a member of the quote structure,
+ *  so we add the indent paramter to both support to dump these two case
+ */
+func DumpLocalReport(report []byte, indent uint32) error {
```

Comment:
I add the indent parameters and the corresponding comments here for DumpLocalReport @jiazhang0 . Thanks. I revise my codes already according to your suggestions.